File: JAVA/Core/StringsIn_java/StringBuffer_and_Builder/Append.java
package StringsIn_java.StringBuffer_and_Builder;

public class Append {
    public static void main(String[] args) {
    StringBuffer str=new StringBuffer("Web Development");
    System.out.println("StringBuilder Function       : str.append(\" in java\")");
    System.out.println("My StringBuffer str          : "+str);
     str.append(" in java");
     System.out.println("String builder_buffer Value  : "+str);
}
}

File: C++/Core/String/Append.cpp
#include <iostream>
using namespace std;
int main(){
  cout<<"---------------------------------"<<endl;
  cout<<"String Method       :str.append(str2)"<<endl;
  string str="Web Development";
  string str2(" In C++");
  cout<<"My string str is    :"<<str<<endl;
  cout<<"My string str2 is   :"<<str2<<endl;
  cout<<"String Method Value :"<<str.append(str2)<<endl;
  cout<<"---------------------------------\n";
  return 0;
}
